src/com/connectsdk/service/DeviceService.java

```diff
@@ -69,9 +69,26 @@
  * All DeviceService objects have a group of capabilities. These capabilities can be implemented by any object, and that object will be returned when you call the DeviceService's capability methods (launcher, mediaPlayer, volumeControl, etc).
  */
 public class DeviceService implements DeviceServiceReachabilityListener, ServiceCommandProcessor {
+
+    /**
+     * Enumerates available pairing types. It is used by a DeviceService for implementing pairing
+     * strategy.
+     */
     public enum PairingType {
+        /**
+         * DeviceService doesn't require pairing
+         */
         NONE,
+
+        /**
+         * In this mode user must confirm pairing on the first screen device (ex. an alert on a TV)
```

Comment:
I believe "In this mode" is unnecessary here, since it's the description of this specific value.

Also, you should be using "e.g.," for examples instead of "ex.".